packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -304,3 +305,56 @@ export function char2Bytes(str: string) {
 export function bytes2Char(hex: string): string {
   return Buffer.from(hex2buf(hex)).toString('utf8');
 }
+
+/**
+ *
+ * @description Convert hex string/UintArray/Buffer to bytes
+ *
+ * @param hex String/Uint8Array/Buffer value to convert to bytes
+ */
+export function hex2Bytes(hex: string | Uint8Array | Buffer): Buffer {
+  if (typeof hex === 'string' && hex.match(/[\da-f]{2}/gi)) {
+    return Buffer.from(hex, 'hex');
+  } else if (!(typeof hex === 'string')) {
+    return Buffer.from(hex);
+  } else {
+    return Buffer.from([0]);
+  }
+}
+
+/**
+ *
+ * @description Converts a number or BigNumber to a padded hexadecimal string
+ * @param val The value that will be converted into a padded hexadecimal string value
+ * @param bitLength The length of bits
+ *
+ */
+export function num2PaddedHex(val: number | BigNumber, bitLength = 8): string {
```

Comment:
Can we add a unit test where the param is a BigNumber?

Reply:
that's a good point, I will add a BigNumber param case